### ISSUE TITLE
ACL 2018 Metadata Correction

### ISSUE THREAD

**Issue by andyweizhao** [acl-org/acl-anthology#100]
Dear Matt,
I am one of the authors in paper entitled “Investigating Capsule Networks with Dynamic Routing for Text Classification”, we report that authorship in metadata is out of sync with the paper:

http://aclweb.org/anthology/D18-1350.bib
http://aclweb.org/anthology/D18-1350.pdf

Please help correct the metadata. Thank you for the correction.

Best Regards,

Wei Zhao

**Comment by mjpost** [comment]
Thanks. To clarify, you would like correct the `.bib` file to reflect the order in the paper, correct? If so, that correct is fine. Please issue a pull request with the corrections you would like made, and references this issue in the PR. The authoritative Anthology source for your paper is [this file](https://github.com/acl-org/acl-anthology/blob/master/import/D18.xml), and it seems the correction you want is [here](https://github.com/acl-org/acl-anthology/blob/918dcc4dba25f07c09fe0807e3f6adbb8d589c6d/import/D18.xml#L5983).

**Comment by andyweizhao** [comment]
Thanks. We have issued an pull request https://github.com/acl-org/acl-anthology/pull/101, but we only changed the D18.xml, the .bib file will be altered accordingly, right?

**Comment by villalbamartin** [comment]
@andyweizhao Indeed, once the XML file is updated, the .bib files will be regenerated to reflect the changes.

**Comment by andyweizhao** [comment]
I see. Thanks.

**Comment by mjpost** [comment]
Okay, this has been merged. 

@villalbamartin, can you please regenerate the website? There may be another correction in the next day, so you may want to delay rebuilding until the weekend.

**Comment by mjpost** [comment]
@villalbamartin That is still the location for hosting the PDFs, so my guess is that the entire tarballs (PDFs and bibs) get dumped in the aclweb.org/anthology directory. For example, [D18-1350 is there](https://aclweb.org/anthology/D18-1350.bib), but it is the original, uncorrected version.

**Comment by andyweizhao** [comment]
@villalbamartin Yes. As matt said, looks there are two versions of bib file. http://aclweb.org/anthology/D18-1350.bib and https://aclanthology.coli.uni-saarland.de/papers/D18-1350/d18-1350.bib. The first one is uncorrected.

**Comment by knmnyn** [comment]
Hi all:

The first is available in the original anthology but that source is
deprecated.
We can edit it manually on the aclweb.org host which the ACL Anthology
Director has singular access to; the volunteer staff does not.

@All: They are still up there for legacy reasons, and a workflow that Matt
and I need to both settle on, can remove or redirect these.  There are
discrepancies between the original, submitted versions from softconf or the
submitted by organizer that are available on aclweb.org.  As of 2018 (last
year), we deprecated those on aclweb.org in favor of the canonicalized
source in the ACL Anthology database generated BibTeX, which is the second
source.  Hope this helps!
